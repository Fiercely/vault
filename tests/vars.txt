VAULT=SECRETS*
MADE=BY=
COMPANY=FIERCELY*
AUTHOR=ILHICAS/
URL=https://www.fiercely.pt
AUTHOR_URL=https://ilhicas.com
