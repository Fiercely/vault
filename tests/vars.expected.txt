VAULT=SECRETS*
MADE=U2FsdGVkX18saI2WsjpiBZZ5WxLnZRWeBOg4RSdFl3Q=
COMPANY=U2FsdGVkX19Ys3YhSHTNuFlfYmHExsPmFDjIMqKsD4Q=
AUTHOR=U2FsdGVkX1//Z8i3BavatDjn5apq1rY+iEVAalIo7O4=
URL=U2FsdGVkX1/lPIzbLvf7NOWomFeHlo8S/s7m0fRPv6Y//FbphAw7XLe90YVLB2FM
AUTHOR_URL=https://ilhicas.com
TAG=LATEST*#!1234"56)7(9)+"'';_|/&%$#==!=:
